spec/acceptance/node_classification_spec.rb

```diff
@@ -36,19 +38,21 @@
         'File',
         "#{TEST_ENVIRONMENT_CODE_DIR}/modules",
         ensure: 'link',
-        target: '/etc/puppetlabs/code/environments/production/modules',
+        target: "#{CODE_DIR_BASE_PATH}/production/modules",
       ),
       declare(
         'node_group',
         TEST_ENVIRONMENT,
         ensure: 'present',
-        parent: 'All Environments',
```

Comment:
This should be `All Environments` b/c that's the group environment groups are derived from. Customers will likely have their groups derived from that too.

Reply:
That group doesn't exist in the container. I'm still working on the best way to get around that. I'll probably switch to a setup manifest in a `before(:all)` that creates it.